scripts/gha/summarize_test_results.py

```diff
@@ -378,7 +380,7 @@ def combine_configs(error_configs, all_configs):
 
 def combine_config(platform, config, config_value, k):
   config_before_combination = config.copy()
-  logging.info("platform: %s; config: %s; config_value: %s", platform, config, config_value)
+  # logging.info("platform: %s; config: %s; config_value: %s", platform, config, config_value)
```

Comment:
nit: should this be removed completely or put back?

Reply:
removed